react-client/src/components/Profile.jsx

```diff
@@ -42,6 +43,7 @@ class Profile extends React.Component {
         <div className='col-md-9'>
           <h1> {userName} Profile</h1>
           <p>{email}</p>
+          <h1>{password}</h1>
```

Comment:
Passwords should never be displayed on a page. Always filter this so that it shows up as a bunch of dots or asterisks.